lib/bankaccount/money_market_account.rb

```diff
@@ -0,0 +1,91 @@
+module BankAccount
+
+	class MoneyMarketAccount < Account
+
+		attr_reader :id, :initial_balance, :account_balance, :frozen_account, :allowed_transactions
+
+		def initialize(id, initial_balance)
+			raise ArgumentError.new "Please enter an initial balance of at least $10,000." unless initial_balance >= 10_000
```

Comment:
You taught me something new! I didn't know you could put an underscore for helping to dilineate place value in integers. That makes it really readable!